### ISSUE TITLE
Example project Shader

### ISSUE THREAD

**Issue by BitBarrel** [rive-app/rive-unity#6]
For the example projects, it is better to use a material with this shader: Unlit->Texture. Using the standard shader for a UI or display simulation doesn't really make sense unless it is setup as an emissive shader with a reflective component.

**Comment by MarcusU** [comment]
Yes! I had such a problem getting Rive animations to work in a simple Image within our 2D app, instead of turning it into a camera or whatever. And the "Assign the RenderTexture to the Material" step in the getting started docs is horrible. So many ways to do that apparently. But what I found that finally worked matches what you said: for the Material, set the shader to Unlit > Texture, then drag the rive RenderTexture into that little box on the Material inspection page. 
Very confusing. 
My animations are still reversed and backwards, and changing the "Rotation" xyz settings don't help. Sometimes the animation does not display at all with a 180 (flip it) setting.
